src/transformers/feature_extraction_utils.py

```diff
@@ -211,6 +211,10 @@ def __init__(self, **kwargs):
                 logger.error(f"Can't set {key} with value {value} for {self}")
                 raise err
 
+    def _set_processor_class(self, processor_class: str):
+        """Sets processor class as an attribute."""
+        self.processor_class = processor_class
```

Comment:
Don't think we should add a public attribute. Like for the tokenizer, it should be private as it's not relevant to the feature extractor, and there should be special logic to load/save it as "processor_class".